data/migrations/20190502153324_group_participants.js

```diff
@@ -21,7 +21,7 @@ exports.up = function(knex, Promise) {
       .defaultTo(false)
       .notNullable();
 
-    table.string("buyin_proof"), defaultTo(null);
+    table.string("buyin_proof").defaultTo('');
```

Comment:
Think we need this for the way we were planning on doing the boolean logic.

table.string("buyin_proof").defaultTo(null);